src/alignArrays.js

```diff
@@ -38,11 +38,15 @@ function longestCommonSubsequence(a, b) {
   const memo = initMatrix(aLength + 1, bLength + 1);
   const solution = initMatrix(aLength + 1, bLength + 1);
 
+  const usedDriftRange = Math.abs(
+    Math.max(Math.max(aLength, bLength) / 10, DRIFT_RANGE),
```

Comment:
DRIFT_RANGE should probably be renamed MIN_DRIFT_RANGE

Reply:
Agreed - done!